code/src/components/Start.js

```diff
@@ -0,0 +1,77 @@
+import React from "react"
+import { useDispatch } from "react-redux"
+
+import { game } from "reducers/game"
+import { startGame } from "reducers/game"
+import styled from "styled-components"
```

Comment:
We're not totally sure but we think that the good practice would be to have the "global" imports together on the top and then having the "local" ones (your components/files) separate after.